src/Header/Header.js

```diff
@@ -1,13 +1,32 @@
 import React from 'react';
+import { Navbar, Nav, Button, Offcanvas, Container, NavDropdown, Form } from 'react-bootstrap';
+import logo from '../Images/company_logo.png'
+import './Header.css'
 
 export const Header = () => {
     return (
-        <header>
-            <h1>
-                Company Logo
-            </h1>
-            <button>About Us</button>
-            <button>Schedule Tour</button>
-        </header>
+        <Navbar bg='light' expand='sm'>
```

Comment:
Make sure you keep double quotes for any html/JSX attributes. This applies to this whole file.